Unit-01/04-flask-crud/templates/new.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+{% block content %}
+
+<h1>Add Yo' Snack!</h1>
+
+  <form action="{{url_for('index')}}" method="POST">
+    <label for="name">Snack Name:</label>
+    <input type="text" name="name">
```

Comment:
If you add a `label` make sure the input it is associate with as an `id` attribute with the same value as the `for` attribute in your label